CREDITS

```diff
@@ -3770,10 +3770,11 @@ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
 ================================================================
 
-github.com/dgrijalva/jwt-go
-https://github.com/dgrijalva/jwt-go
+github.com/golang-jwt/jwt
+https://github.com/golang-jwt/jwt
 ----------------------------------------------------------------
 Copyright (c) 2012 Dave Grijalva
+Copyright (c) 2021 jwt-go maintainers
```

Comment:
is this manually edited? @emirisman

Reply:
It is the current LICENSE file https://github.com/golang-jwt/jwt/blob/main/LICENSE

Reply:
oh my bad, copied the wrong string